src/com/epam/Exceptions/helpers/UniversityHelper.java

```diff
@@ -0,0 +1,248 @@
+package com.epam.Exceptions.helpers;
+
+import com.epam.Exceptions.FacultyAbsenceException;
+import com.epam.Exceptions.GroupAbsenceException;
+import com.epam.Exceptions.StudentAbsenceException;
+import com.epam.Exceptions.SubjectAbsenceException;
+import com.epam.Exceptions.models.*;
+
+import java.util.Random;
+
+public class UniversityHelper {
+
+    public static University creteTheUniversity() {
+        String mathematics = "Mathematics";
+        String english = "English";
+        String armenian = "Armenian";
+        String logic = "Logic";
+        String sociology = "Sociology";
+        String physics = "Physics";
+
+        String[] subjectsOfGroup701 = new String[]{physics, sociology, armenian};
+        String[] subjectsOfGroup702 = new String[]{mathematics, logic, sociology};
+        String[] subjectsOfGroup501 = new String[]{english, logic, armenian};
+        String[] subjectsOfGroup502 = new String[]{english, sociology, physics};
+
+        Mark.MarkBuilder markBuilder = new Mark.MarkBuilder();
+        markBuilder.subject(physics);
+        Mark physicsA = markBuilder.value(5).build();
```

Comment:
You may combine the call of subject() and value() methods into one line:
markBuilder.value(5).subject(..).build();